### PR TITLE
Create function for Mapping username-userId 

### PR DESCRIPTION
**Pull request by sanyogita14**
Function `toUserId` receives the username and returns the corresponding userId for that user.
Function `toUsername` does the opposite - userId received as a parameter and the corresponding username returned.

### CODE REVIEW

**Review by swarajpure — DISMISSED** [review]
Looks good to me!
Thanks for this. Very much needed!

File: utils/idMapping.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const { fetchUser } = require('../models/users')
+
+/**
+ * Used for receiving userId when providing username
+ *
+ * @param username {String} - username of the User.
+ * @returns id {String} - userId of the same user
+ */
+const toUserId = async (username) => {
```
Comment (ankurnarkhede) [inline utils/idMapping.js:9]: Can we simply name the functions as getUserId and getUsername?

toUserId seems fine when its a class method, but here, its a function.

File: utils/idMapping.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const { fetchUser } = require('../models/users')
```
Comment (ankurnarkhede) [inline utils/idMapping.js:1]: Can we change the file name as users.js, so that in future, we can add other utilities related to user data?

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]
Need unit tests.

**Comment by sanyogita14** [comment]
> Need unit tests.

Can I make a separate PR for the tests?

File: utils/idMapping.js (hunk 2 above)
Comment (sanyogita14) [reply utils/idMapping.js:1]: yeah ok

**Comment by ankurnarkhede** [comment]
> > Need unit tests.
> 
> Can I make a separate PR for the tests?

@sanyogita14 , we should add tests to the same PR if possible.

**Review by swarajpure — APPROVED** [review]
@ankurnarkhede Since the changes which you had mentioned were made, I'm merging this PR now. 
It's a small PR but this PR is critical for all the backend devs working on crypto tasks for public release on 14 March

I've created an issue for unit tests and assigned to @sanyogita14 #228

**Pull request merged**